src/Common/src/System/Net/ContextAwareResult.cs

```diff
@@ -153,7 +153,7 @@ internal ExecutionContext ContextCopy
                 ExecutionContext context = _context;
                 if (context != null)
                 {
-                    return context.CreateCopy();
+                    return context; // No need to copy on CoreCLR; ExecutionContext is immutable
```

Comment:
More specifically, not on the full framework, right?  I believe so, but... @weshaggard? @davidsh?

CreateCopy isn't exposed in the contract, so I'm not sure what else we could do as long as we want to use the real ExecutionContext (which we should really want to do ;)